### ISSUE TITLE
I'd like to just have the article, instead of the article appended with the word.

### ISSUE THREAD

**Issue by ChrisTolmeijer** [tandrewnichols/indefinite#13]
I'd like to just have the article, instead of the article appended with the word, because I want to encapsulate the word itself into different JSX tags. I'm proposing the following. Can make a pull request:

```javascript
exports.capitalize = (article, word, opts) => {
  if (opts.capitalize) {
    article = `${article.charAt(0).toUpperCase()}${article.slice(1)}`;
  }

  let result = `${article} ${word}`;

  if(opts.articleOnly) {
     result = article;
  }

  return result;
};
```

**Comment by tandrewnichols** [comment]
Yeah, seems reasonable. Don't forget to update docs/tests to reflect.

**Comment by mrseanryan** [comment]
I think this issue can be closed

**Comment by tandrewnichols** [comment]
Yeah, good call. This was published in indefinite@2.3.0.
